docs/src/markdown-pages/aptitudes/ui.md

```diff
@@ -0,0 +1,6 @@
+---
+name: "UI"
+links_go: "UIClient"
+links_node: "UIService"
+---
+Allows Loops to receive a notification whenever the user interacts with UI elements in Olive Helps
```

Comment:
Very minor - you have "."s at the end of some of these descriptions and not others.